### PR TITLE
Mod 0 syllabus

### PR DESCRIPTION
**Pull request by rtillies**

### CODE REVIEW

File: Mod0/syllabus.md [hunk 1]
```diff
@@ -4,25 +4,33 @@ title:      Syllabus
...
 
-- **Typing Speed:** It is recommended that students have an average typing speed of at least 50 words per minute before beginning their journey at Turing. You can test your current typing speed [here](https://www.typing.com/student/tests). If you find that you are not currently typing 50 words per minute or faster, this does not mean you cannot be successful at Turing, but it does mean you will need to improve this skill. Typing quickly and accurately without looking at the keyboard will make many things easier throughout your time at Turing.
+- **Typing Speed:** It is recommended that students have an average typing speed of at least 50 words per minute before beginning their journey at Turing. You can test your current typing speed [here](https://www.typing.com/student/tests). While not a requirement for the program, aim for a goal of typing 50 words per minute or faster. Typing quickly and accurately without looking at the keyboard will make many things easier throughout your time at Turing. Also, you will find that typing code is different than typing "regular" text (e.g. a letter or essay).
```
Comment (memcmahon) [inline Mod0/syllabus.md:21]: Here is the 'code' typing site: https://typing.io/lesson/csharp/asp.net/CSharpCodeParser.cs/1

File: Mod0/syllabus.md [hunk 2]
```diff
@@ -77,23 +80,16 @@ There will be no “gotchas”. The rubric that follows communicates what we car
 ## Required Supplies
 This may be your first time learning and/or working in a remote setting. There are a few key tools we recommend to set yourself up for success as a remote student. 
 
-- **External Monitor/Display:** There are plenty of great used options available on eBay, Facebook Marketplace, Craigslist, etc. All you really need is something that fits in your workspace and can connect to your MacBook! If you are unable to acquire an external monitor prior to Mod 1, please DM the Mod 0 Lead on Slack and we can discuss other options.
-- **USB-C to HDMI Cable:** This cable will allow you to connect your MacBook to your external monitor/display. If you are purchasing a new monitor, it will likely come with the cable to you, but you may need to aquire one.
+- **External Monitor/Display:** You will receive an external monitor along with your laptop. You will appreciate the additional screen space using all the tools as you navigate your new career. An HDMI-to-HDMI cable (i.e. full-size HDMI plug on both ends) is included in the monitor box to connect your laptop to the monitor.
```
Comment (memcmahon) [inline Mod0/syllabus.md:83]: Is this confirmed?  🤞

File: Mod0/syllabus.md [hunk 3]
```diff
@@ -77,23 +80,16 @@ There will be no “gotchas”. The rubric that follows communicates what we car
...
 - **Dedicated Workspace:** It is crucial to have a workspace that has access to a reliable internet connection and is both quiet and free from distractions. Take time before Mod 0 to build your workspace and experiment with different setups until you find a configuration that works for you.
-- **Notebook + Writing Utensils:** While you will be working on your laptop while you code, work on projects, etc., we recommend using a physical notebook to take notes during class and workshops.
+- **Notebook + Writing Utensils:** While you will be working on your laptop while you code, work on projects, etc., we recommend using a physical notebook to take notes during class and workshops. There are apps available
```
Comment (memcmahon) [inline Mod0/syllabus.md:85]: I think this sentence needs to be completed :)

File: Mod0/syllabus.md [hunk 4]
```diff
@@ -77,23 +80,16 @@ There will be no “gotchas”. The rubric that follows communicates what we car
...
-- **Notebook + Writing Utensils:** While you will be working on your laptop while you code, work on projects, etc., we recommend using a physical notebook to take notes during class and workshops.
+- **Notebook + Writing Utensils:** While you will be working on your laptop while you code, work on projects, etc., we recommend using a physical notebook to take notes during class and workshops. There are apps available 
 
 ## Optional Supplies
 - **Headphones with Microphone:** While your Turing-issued laptop has built-in speakers and a microphone, if your workspace has a lot of ambient noise, it might be a good idea to invest in headphones + microphone. 
-- **USB-C Hub:** Your Turing-issued laptop only has 2 ports. You'll need one for your charger and one for your external monitor. If you require more ports, a USB-C Hub is a great idea!
+- **USB-C Hub:** Your Turing-issued laptop has a limited number of ports. One is for your charged and another is for your external monitor. If you require more ports, a USB-C Hub is a great idea!
```
Comment (memcmahon) [inline Mod0/syllabus.md:89]: one is for your charger

**Review by memcmahon — APPROVED** [review]

File: Mod0/syllabus.md [hunk 5]
```diff
@@ -77,23 +80,16 @@ There will be no “gotchas”. The rubric that follows communicates what we car
...
 - **External Keyboard:** You may benefit from using a different keyboard than the integrated one in your laptop, especially if you are using a laptop stand. If you plan to use a wired keyboard, you'll likely need the USB-C hub mentioned above to connect to your keyboard in addition to your charger and monitor.
 
-
-<!-- ## Resources
-- Turing Lingo
-- JavaScript Cheat Sheet
-- Ruby Cheat Sheet -->
-
 ## Accommodations
 We do our best to work with students who need additional accommodations while attending Turing. If this applies to you, please review [this form](https://frontend.turing.edu/documents/reasonable_accommodation_request_form.pdf) prior to starting Mod 0 at Turing and communicate with the Mod 0 Lead.
```
Comment (zoefarrell) [inline Mod0/syllabus.md:94]: This is very small thing, but I would change from Mod 0 lead to one of your instructors.

File: Mod0/syllabus.md (hunk 2 above)
Comment (rtillies) [reply Mod0/syllabus.md:83]: @memcmahon Ramiro spoke about it at retro yesterday. I may be on a trial basis, but I think it's confirmed.

File: Mod0/syllabus.md (hunk 3 above)
Comment (rtillies) [reply Mod0/syllabus.md:85]: Nope, that was planned ;-) Kidding, I'm on it.

File: Mod0/syllabus.md (hunk 5 above)
Comment (rtillies) [reply Mod0/syllabus.md:94]: Good call out. The original intro paragraph identified another Turing employee as the Mod 0 Lead, and all roads led to her.

**Pull request merged**